library/src/test/java/bio/terra/landingzone/stairway/flight/create/resource/step/ValidateLandingZoneParametersStepTest.java

```diff
@@ -68,19 +73,4 @@ void testFailedValidation() throws InterruptedException {
         stepResult.getException().get().getClass(), equalTo(InvalidInputParameterException.class));
     verify(validator, times(1)).validate(any());
   }
-
-  @ParameterizedTest
```

Comment:
Now that the `ParametersResolver` isn't passed in as a parameter, this test didn't seem very valuable to me